apps/bbsync/query.py

```diff
@@ -143,6 +131,15 @@ def generate_source(self):
         source = source if source else None
         self.add_conditionally("source", source)
 
+    def generate_string(self, flaw_attribute, srtnotes_attribute):
+        """
+        generate given string attribute
+        generic generator for string attributes with no special handling
+        """
+        self.add_conditionally(
+            srtnotes_attribute, getattr(self.flaw, flaw_attribute) or None
+        )
```

Comment:
So lets say in `cvss3` case where this field is empty, we call `getattr(self.flaw, "cvss3")` we got empty string `""` which evaluates to `False` in instead of that the `None` value is passed. Is there any reason for it? We have the empty representation, why do we want to pass over different empty representation? :thinking: Maybe I am missing something.

Reply:
Ahhh, ok, I see now. Thanks for the explanation. LGTM then.